python/tvm/autotvm/tuner/xgboost_cost_model.py

```diff
@@ -327,14 +329,9 @@ def _get_feature(self, indexes):
 
         if need_extract:
             pool = self._get_pool()
-            # If we are forking, we can pass arguments in globals for better performance
-            if multiprocessing.get_start_method(False) == "fork":
-                feas = pool.map(self.feature_extract_func, need_extract)
-            else:
-                args = [(self.space.get(x), self.target, self.task) for x in need_extract]
-                feas = pool.map(self.feature_extract_func, args)
+            feas = pool.map_with_error_catching(self.feature_extract_func, need_extract)
             for i, fea in zip(need_extract, feas):
-                fea_cache[i] = fea
+                fea_cache[i] = fea.value
```

Comment:
```suggestion
                fea_cache[i] = fea.value if fea.status == StatusKind.COMPLETE else None
```
let's be cautious with potential failure in rare cases

Reply:
done